js/filters.js

```diff
@@ -143,14 +143,22 @@ function TypeaheadFilter(elm) {
   Filter.call(this, elm);
 
   var key = this.$body.data('dataset');
+  var allowText = this.$body.data('allow-text') !== undefined;
   var dataset = typeahead.datasets[key];
-  this.typeaheadFilter = new typeaheadFilter.TypeaheadFilter(this.$body, dataset);
+  this.typeaheadFilter = new typeaheadFilter.TypeaheadFilter(this.$body, dataset, allowText);
```

Comment:
Okay, so the presence of `data-allow-text` signifies `true` regardless of what it's set to and its absence is the `false` condition for `allowText`.  Just wanted to be sure!